code/game/machinery/computer/pod.dm

```diff
@@ -304,14 +304,18 @@
 		visible_message("Cannot locate any mass driver of that ID. Cancelling firing sequence!")
 		return
 
-	if(teleporter_dest)
-		for(var/obj/structure/deathsquad_tele/D in world)
-			if(D.z != src.z)	continue
-			if(D.id_tag == ident_tag)
-				D.icon_state = "tele1"
-				D.ztarget = teleporter_dest
-				D.density = 1
-
+	var/spawn_marauder[] = new()
```

Comment:
Ehm, sort of. It's a copy-paste from the specops shuttle code, which was changed to make sure the Marauders work.